packages/optimistic-oracle/index.js

```diff
@@ -119,8 +129,14 @@ async function Poll(callback) {
       errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 5,
       // Default to 10 seconds in between error re-tries.
       errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 10,
+      // Common price feed configuration passed along to all those constructed by proposer.
+      commonPriceFeedConfig: process.env.OOPRICE_FEED_CONFIG
```

Comment:
Why the name difference, `OOPRICE_FEED_CONFIG` vs. `commonPriceFeedConfig`? Should this be `COMMON_PRICE_FEED_CONFIG`?

Reply:
Yeah agreed, and changed.